README.md

```diff
@@ -1 +1,251 @@
+1-HTML.
+2-CSS.
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+-Added index.html file.
+-Added style.css file.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+ comming soon
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1.browser.
+2.code editor.
+3.Git smc.
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this commands
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/jalal1379/set-up-hello.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+Run this command:
+
+```sh
+  cd my-project
+  nbm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extention.
+
+### Run tests
+
+comming soon.
+
+### Deployment
+
+comming soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 Mohammad Jalal Ahmai
+
+- GitHub: [@githubhandle](https://github.com/jalal1379)
+- Twitter: [@twitterhandle](https://twitter.com/JalalAh78951382)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/jalal-ahmadi-58312b270/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- [ ] Add photo.
+- [ ] Add footer.
+- [ ] Add link.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project just give it to start.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank Microverse to give me a chance.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
```

Comment:
- [ ] The FAQ section of the README file is optional and it would be great if you can ad one or two questions there for anybody visiting to know about project or probably remove it if you didn't have any question to add. In this case I would advice you remove it rather than leaving it empty :+1: